cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -163,7 +186,9 @@ def expand_date(df: pyspark.sql.DataFrame) -> pyspark.sql.DataFrame:
         .withColumn("Day", F.dayofmonth(df.Date))
     )
 
-
+# %%
+# Now, we define a function that will remove outliers from the data.
+# It helps count the number of days before or after a special event (such as a promo or a holiday)
```

Comment:
What outliers are we removing?
In "It helps ...", what is it?

Also, can you please remove "we define a function..."? It's kinda everywhere. Just follow the format used in the previous data-processing steps.

Reply:
"Number of days before or after" meaning? From when?